integration-test/IntegrationTest/ThutSpec.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TupleSections     #-}
+module IntegrationTest.ThutSpec where
+
+import           Frost.Effects.Thut
+import           FrostError
+
+import           Polysemy
+import           Polysemy.Error
+
+import           Data.Function      ((&))
+import           Test.Hspec
+
+spec :: Spec
+spec =
+  describe "Frost.Effects.Thut runThutIO" $
+    it "should execute thut:passthrough" $ do
+      pendingWith "make it work"
```

Comment:
Looks like this fails due to: https://github.com/NixOS/nixpkgs/issues/70787

Reply:
i can not event get it running with ghc865 :)

Reply:
I wasn't able to make it work :-( Help & ideas are welcome